File: BE/src/main/java/com/self/roomescape/entity/Cafe.java
package com.self.roomescape.entity;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.*;

@Entity
@Getter
@Setter
@ToString
@Table(name = "cafe")
public class Cafe {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int cid;

//    private int oid;

    private String cname;
    private String cphone;
    private String url;
    private String time;
    private String img;
    private String location;
    private String lat;
    private String lon;
    @Column(name = "do")
    private String island;
    private String si;
}

File: AOS/app/src/main/java/com/ssafy/moabang/data/model/dto/Cafe.kt
package com.ssafy.moabang.data.model.dto

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.maps.android.clustering.ClusterItem
import kotlinx.parcelize.IgnoredOnParcel
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity
data class Cafe (
    @PrimaryKey var cid : Int,
    var cname : String?,
    var cphone : String?,
    var url : String?,
    var time : String?,
    var img : String?,
    var location : String?,
    var lat : String?,
    var lon : String?,
    var island : String?, // 도(섬 도 島)
    var si : String?, // 시
): Parcelable{
    @IgnoredOnParcel
    var distance : Double = 0.0
}
